tailormap-components/projects/core/src/lib/user-interface/geometry-confirm-buttons/geometry-confirm-buttons.component.ts

```diff
@@ -0,0 +1,55 @@
+import {
+  Component,
+  OnDestroy,
+  OnInit,
+} from '@angular/core';
+import { GeometryConfirmService } from './geometry-confirm.service';
+import { takeUntil } from 'rxjs/operators';
+import { Subject } from 'rxjs';
+import { ScreenCoordinate } from '../models';
+
+@Component({
+  selector: 'tailormap-geometry-confirm-buttons',
+  templateUrl: './geometry-confirm-buttons.component.html',
+  styleUrls: ['./geometry-confirm-buttons.component.css'],
+})
+export class GeometryConfirmButtonsComponent implements OnInit, OnDestroy {
+
+  private destroyed = new Subject();
+  public position: ScreenCoordinate;
+  public visible = false;
+
+  constructor(private geometryConfirmService: GeometryConfirmService) {
+    this.position = {
+      left: 0,
+      top: 0,
+    };
+    this.geometryConfirmService.positionChanged$
+      .pipe(takeUntil(this.destroyed)).subscribe(value => {
+      this.position = {
+        left: value.left + 5,
```

Comment:
5 is some margin & 50 is the height of the buttons? Maybe its nice to make variables for these values so that is clear what they mean